damnit/gui/table.py

```diff
@@ -377,6 +378,9 @@ def text_item(self, value, display=None):
     def image_item(self, png_data: bytes):
         item = self.itemPrototype().clone()
         item.setData(self.generateThumbnail(png_data), role=Qt.DecorationRole)
+        item.setToolTip(
+            f'<img src="data:image/png;base64,{b64encode(png_data).decode()}">'
```

Comment:
Am I correct in thinking that we would need to specify `width` and `height` to control the size of the tooltip?

Reply:
I assume all thumbnails are saved as 300x300 px, right?
but yea, it does not hurt to enforte it here

Reply:
Yes, I'm just asking out of curiosity in case we want to change it in the future. I think leaving them at full resolution is fine though, it looks decently sized when displayed.